website/src/bok/DocTagDocListPage/index.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import clsx from 'clsx';
+import Link from '@docusaurus/Link';
+import {
+  PageMetadata,
+  HtmlClassNameProvider,
+  ThemeClassNames,
+  usePluralForm,
+} from '@docusaurus/theme-common';
+import Translate, {translate} from '@docusaurus/Translate';
+import Layout from '@theme/Layout';
+import SearchMetadata from '@theme/SearchMetadata';
+// Very simple pluralization: probably good enough for now
+function useNDocsTaggedPlural() {
+  const {selectMessage} = usePluralForm();
+  return (count) =>
+    selectMessage(
+      count,
+      translate(
+        {
+          id: 'theme.docs.tagDocListPageTitle.nDocsTagged',
+          description:
+            'Pluralized label for "{count} docs tagged". Use as much plural forms (separated by "|") as your language support (see https://www.unicode.org/cldr/cldr-aux/charts/34/supplemental/language_plural_rules.html)',
+          message: 'One doc tagged|{count} docs tagged',
+        },
+        {count},
+      ),
+    );
+}
+function DocItem({doc}) {
+  return (
+    <article className="margin-vert--lg">
+      <Link to={doc.permalink}>
+        <h3>{doc.title}</h3>
+      </Link>
+      {doc.description && <p>{doc.description}</p>}
+    </article>
+  );
+}
+
+const categoryStubs = [ 'Risks', 'Roles', 'Activities', 'Training'];
+
+function sortDocs(items, exclude) {
+  const out = {}
+
+  items
+    .filter(i => i !== exclude)
+    .forEach(i => {
+    categoryStubs.forEach(c => {
+      if (i.permalink.indexOf(c) > -1) {
+        const list = out[c];
+        if (!list) {
+          out[c] = [];
+        }
+
+        out[c].push(i);
+      }
+    })
+
+    return out;
+  })
+
+  return out;
+
+}
+
+function mainDoc(tag) {
+  const index = tag.label.indexOf(" (");
+  if (index > -1) {
+    const tagWithoutBracket = 'CIO/CTO'; // tag.label.substr(0, index);
```

Comment:
Nice save :)

Reply:
This doesn't really do anything in this PR, but I'll address this when we add the body-of-knowledge set of documents, so I'm going to leave this for now.